radicle-cli/tests/framework/mod.rs

```diff
@@ -8,6 +8,9 @@ use snapbox::cmd::{Command, OutputAssert};
 use snapbox::{Assert, Substitutions};
 use thiserror::Error;
 
+/// Error lines in the CLI are prefixed with this string.
+const ERROR_PREFIX: &str = "==";
```

Comment:
I might need to update my `rad id` output, since I was using `=== <header> ===`. Any suggestions on what the headers should look like?

Reply:
Yeah indeed I think that should change. I would keep it simple for now and just have headers with a different color to the body text, plus bold or underline. At some point maybe we'll spice things up a bit.